util-core/src/main/scala/com/indix/utils/core/UrlUtils.scala

```diff
@@ -41,7 +41,7 @@ object UrlUtils {
     */
   def decode(url: String) = URLDecoder.decode(url, "UTF-8")
 
-
+  def encode(url: String) = URLEncoder.encode(url, "UTF-8").replace("+", "%20")
```

Comment:
@Sasikanth3 you might have to add the scala docs comments as done for other methods.